lib/src/base-styles/_nx-divider.scss

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.nx-divider {
+  background-color: var(--nx-swatch-indigo-90);
+  border-radius: 1px;
```

Comment:
I'm not sure a border radius here is part of the spec.  The divider in sketch seems squared-off no matter how far I zoom in.  Not to mention, wouldn't this need to be 0.5?

Reply:
> I'm not sure a border radius here is part of the spec. The divider in sketch seems squared-off no matter how far I zoom in. Not to mention, wouldn't this need to be 0.5?

Removed here: 1c8d806169e9f3fcf770a7d8dc42b9e9ab95436c
The sketch file shows that there was a radius of 1, so I initially had it as 1. But for something with a `height` of `1px`, like you said, it's really difficult to see the `border-radius` anyway.